sql-plugin/src/main/java/com/nvidia/spark/rapids/UnsafeRowToColumnarBatchIterator.java

```diff
@@ -43,7 +44,7 @@
  * be generated based off of the schema.
```

Comment:
nit: Could we update the docs a bit? Explain what is happening with InternalRow?

Reply:
I renamed the classes from UnsafeRow -> InternalRow to make this a bit clearer and added a small blurb about the rows being converted to UnsafeRow before going to ColumnarBatch.